tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/sealed-array-semantics-negative.bal

```diff
@@ -27,7 +27,7 @@ function accessInvalidIndexOfSealedArrayUsingKeyword() returns string {
 }
 
 function initializeInvalidSizedSealedArray() {
-    boolean[4] sealedArray1 = [true, true, false];
+    //boolean[4] sealedArray1 = [true, true, false]; // this is valid now
```

Comment:
Why are we commenting these out instead of removing them?

Reply:
Let me remove. I wanted to avoid rewriting every line number of the java side asset on message and line number.